userguide/aviate/aviate-usage-tutorial.adoc

```diff
@@ -0,0 +1,277 @@
+= Usage Tutorial - AI Use Case
+
+include::{sourcedir}/aviate/includes/aviate-card.adoc[]
+
+== Introduction
+
+This tutorial shows how to use the Aviate APIs together with open-source Kill Bill APIs to address usage-based billing scenarios.
+
+Kill Bill already supports sophisticated catalogs, usage-based plans, and automated invoicing. Aviate extends these capabilities with:
+
+* A scalable metering system for capturing usage data points.
+* Wallet functionality integrated with Kill Bill to manage credit pools and balances.
+* Advanced product and plan management, also integrated with Kill Bill.
+
+In this example, we simulate an AI use cases where input tokens are recorded as usage and billed periodically via paid credits.
+
+== Prerequisites
+
+You will need:
+
+* A running Kill Bill with the Aviate plugin installed.
+* A valid API access tokens for all Aviate APIs calls - referred to is `ID_TOKEN`
+
+== Setup: Configuring a Tenant
+
+=== Step 1: Create a tenant
+
+We start by creating a tenant `usage-scenario`.
+
+Use the standard Kill Bill API: https://killbill.github.io/slate/tenant.html#create-a-tenant
```

Comment:
```suggestion
Use the standard Kill Bill API: https://apidocs.killbill.io/tenant.html#create-a-tenant
```

Reply:
I have not changed those because this is the link I get when I click from our doc - and editing all links manually afterwards seems error prone.